File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{ 
    // Get n = 1-8 
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    
    // Print out that many hashtag
    int i = 1;
    
    while (i <= n) //create another line of hashtag if i < n
    {   
        for (int k = n - i; k > 0; k--) //making the left-hand side of the pyramid right-aligned 
        
        {
            printf(" "); 
        }
        
        for (int j = 0; j < i; j++) //print #

        { 
            printf("#");
        }
        
        printf("  "); //the space in between
        
        for (int j = 0; j < i; j++)

        { 
            printf("#");
        }
        
        printf("\n"); //be careful where to put this line; will affect where to start a new line
        i++;
        
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

height = 0
row = 1

while height < 1 or height > 8:
    height = get_int("Height: ")

while row <= height:
    i = row + 1
    while i <= height:
        print(" ", end='')  # end='' = print in the same line
        i += 1
    for j in range(row):
        print("#", end='')
    print("  ", end='')
    for k in range(row):
        print("#", end='')
    print("")
    row += 1
    